problem_builder/tests/integration/test_questionnaire.py

```diff
@@ -115,17 +112,9 @@ def submit_answer_and_assert_messages(mcq1_answer, mcq2_answer, item_feedback1,
             mcq1_tips = mcq1.find_element_by_css_selector(".choice-tips .tip p")
             mcq2_tips = mcq2.find_element_by_css_selector(".choice-tips .tip p")
 
-            self.assertTrue(mcq1_feedback.is_displayed())
```

Comment:
It's not that I just removed assertions :) Feedback is now hidden (so these assertions are not needed), and tips are displayed without the need to click on them.